.github/workflows/triage.yml

```diff
@@ -0,0 +1,53 @@
+name: Triage new issues
+
+on:
+  issues:
+    types: [ opened ]
+
+env:
+  GITHUB_TOKEN: ${{ secrets.PROJECT_BOARD_AUTOMATION }}
+  ORGANIZATION: ethereum
+  PROJECT_NUMBER: 43
+  DRY_RUN: true
+
+jobs:
+  triage_issues:
+    runs-on: ubuntu-latest
+    if: join(github.event.issue.labels) == ''
+    steps:
+      - name: Retrieve project data
+        env:
+          ORGANIZATION: ${{ env.ORGANIZATION }}
+          PROJECT_NUMBER: ${{ env.PROJECT_NUMBER }}
+        run: |
+          gh api graphql --raw-field query='
+            query($org: String!, $number: Int!) {
+              organization(login: $org) {
+                project(number: $number) {
+                  columns(first: 10) {
+                    nodes {
+                      id,
+                      name
+                    }
+                  }
+                }
+              }
+            }' --raw-field org=$ORGANIZATION --field number=$PROJECT_NUMBER >> project_data.json
+          echo 'COLUMN_ID='$(jq '.data.organization.project.columns.nodes[] | select(.name == "Triage") | .id' project_data.json) >> $GITHUB_ENV
+          echo 'COLUMN_NAME='$(jq '.data.organization.project.columns.nodes[] | select(.name == "Triage") | .name' project_data.json) >> $GITHUB_ENV
+      - name: Add issue#${{ github.event.issue.number }} to Triage column
+        env:
+          ISSUE_ID: ${{ github.event.issue.node_id }}
+          PROJECT_NUMBER: ${{ env.PROJECT_NUMBER }}
+        run: |
+          if [ ${{ env.DRY_RUN }} ]; then
+            echo "Adding issue: ${{ github.event.issue.number }} to column $COLUMN_NAME in project $PROJECT_NUMBER"
+          else
+            gh api graphql --raw-field query='
+              mutation($column: ID!, $issue: ID!) {
+                addProjectCard(input: {
+                  projectColumnId: $column,
+                  contentId: $issue
+                }) { clientMutationId }
+              }' --raw-field column=$COLUMN_ID --raw-field issue=$ISSUE_ID --silent
```

Comment:
What will happen if the issue is already on the board? Either in Triage or in another column? Will this fail or just remove it from the other column?

Reply:
Hum... good question. I believe that it will attempt to move it anyway. But the action is only triggered when the issue is open (and has no labels), so in theory, it can only be triggered once per issue.

Reply:
ok, I guess we'll see how it behaves when we merge. Some weird effects will likely be easy to notice and fix.

Reply:
As the job only runs when an issue is opened, I believe we will not have such a problem. However, if we also trigger the job for reopening, we will have a problem when attempting to add an already existing issue in the board because of the use of the `addProjectCard` method.

I tested here, and when an issue is reopened, the issue can be moved automatically to a column if that column has set the `Reopened` option in the board settings. And if that is the case, and we also allow the workflow to be triggered by reopening, the action will fail.

As a reopened issue may require a new evaluation of the task, it may be good to change that command and support adding new issues to triage or move them to triage if they are reopened.

Reply:
Right, this should run also on reopening as long as the issue is still untriaged. It does happen that a someone closes something by accident and then reopens it. Having it drop from Triage in that case is undesirable. But that's something for a followup PR. We can always fix these things manually when we notice.

Reply:
Indeed. I will modify it to add another step for reopening ;)